src/test/java/com/google/api/codegen/testdata/discogapic/java/java_fakebilling.v1.json.baseline

```diff
@@ -382,54 +381,55 @@ public final class BillingAccountsGetHttpRequest {
     this.prettyPrint = prettyPrint;
     this.quotaUser = quotaUser;
     this.userIp = userIp;
+    this.fooRepeated = fooRepeated;
+  }
 
-    ImmutableListMultimap.Builder pathParams = ImmutableListMultimap.builder();
-    ImmutableListMultimap.Builder queryParams = ImmutableListMultimap.builder();
+  private BillingAccountsGetHttpRequest() {}
 
-    if (name != null) {
-      pathParams.put("name", name.toString());
+  public void populateFieldsInMap(Set<String> fieldNames, Map<String, List<String>> map) {
+    if (fieldNames.contains("name")) {
+      List<String> values = new LinkedList<>();
+      values.add(name);
+      map.put("name", values);
```

Comment:
I think it might be nice to make this denser:

`map.put("name", Lists.newArrayList(name));`

Or maybe we want to make it immutable...

`map.put("name", Collections.singletonList(name));`

Although if we're making it immutable, we could just return the result instead of having to accept it as a parameter.